src/main/java/com/fundation/logic/model/Criteria.java

```diff
@@ -19,199 +19,199 @@
   */
 
  public class Criteria {
-     private String path;
-     private String fileName;
-     private String extension;
-     private Boolean fileHidden = false;
-     private Boolean fileReadOnly = false;
-     private Float sizeLowerLimit;
-     private Float sizeUpperLimit;
-     private Date creationDateLL;
-     private Date creationDateUL;
-     private Date accessDateLL;
-     private Date accessDateUL;
-     private Date modificationDateLL;
-     private Date modificationDateUL;
+     private String criteriaPath;
+     private String criteriaFileName;
+     private String criteriaExtension;
+     private Boolean criteriaFileHidden;
+     private Boolean criteriaFileReadOnly;
+     private Float criteriaSizeLowerLimit;
+     private Float criteriaSizeUpperLimit;
+     private Date criteriaCreationDateLL;
+     private Date criteriaCreationDateUL;
+     private Date criteriaAccessDateLL;
+     private Date criteriaAccessDateUL;
+     private Date criteriaModificationDateLL;
+     private Date criteriaModificationDateUL;
```

Comment:
LL, UL is not compatible with the convention code. Review it and their getters and setters method names.